src/utils/addServer.test.js

```diff
@@ -0,0 +1,61 @@
+import addServers from './addServers';
+
+describe('addServers', () => {
+  const servers = {
+    dicomWeb: [
+      {
+        name: 'DCM4CHEE',
+        wadoUriRoot: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/wado',
+        qidoRoot: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/rs',
+        wadoRoot: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/rs',
+        qidoSupportsIncludeField: true,
+        imageRendering: 'wadors',
+        thumbnailRendering: 'wadors',
+        requestOptions: {
+          requestFromBrowser: true,
+        },
+      },
+    ],
+    oidc: [
+      {
+        authority: 'http://127.0.0.1/auth/realms/ohif',
+        client_id: 'ohif-viewer',
+        redirect_uri: 'http://127.0.0.1/callback',
+        response_type: 'code',
+        scope: 'openid',
+        post_logout_redirect_uri: '/logout-redirect.html',
+      },
+    ],
+  };
+
+  let store = {};
+
+  beforeEach(() => {
+    store = {
+      dispatch: jest.fn(),
+    };
+  });
+
+  test('should be able to add a server and dispatch to the store successfuly', () => {
+    addServers(servers, store);
+    expect(store.dispatch).toBeCalled();
```

Comment:
If this is the only side effect of the method call, you may want to assert the arguments `dispatch` was called with

Reply:
Awesome catch! Thanks!

Reply:
done